spec/support/is_likeable.rb

```diff
@@ -0,0 +1,24 @@
+shared_examples "it is likeable" do
+  before(:each) do
+    # Possibly a horrible hack.
```

Comment:
Nah, seems pretty reasonable to me. I think the default factory for all our models is the same as the model name - if not, it probably should be!